tests/connection/connection.cfg

```diff
@@ -1,4 +0,0 @@
-[imc]
```

Comment:
@britthouser so i looked this up. I think a better way to do this, is not to add this in `.gitignore`, and ask git to not check this for changes separately..